### PR TITLE
Add PyCharm with Anaconda plugin 2020.1

### PR DESCRIPTION
**Pull request by sicruse**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/pycharm-with-anaconda-plugin.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  sha256 'db98e30cc1c16c6cc68ca89949b4eb55dd36e03cbd2b7452b9be06a6ed0c16f4'
+
+  url "https://download.jetbrains.com/python/pycharm-professional-anaconda-#{version.before_comma}.dmg"
+  appcast 'https://data.services.jetbrains.com/products/releases?code=PCC&latest=true&type=release'
+  name 'Jetbrains PyCharm with Anaconda plugin'
+  homepage 'https://www.jetbrains.com/pycharm/promo/anaconda'
+
+  auto_updates true
+
+  app 'PyCharm with Anaconda plugin .app'
```
Comment (vitorgalvao) [inline Casks/pycharm-with-anaconda-plugin.rb:12]: Is the space at the end correct?

File: Casks/pycharm-with-anaconda-plugin.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  uninstall_postflight do
+    ENV['PATH'].split(File::PATH_SEPARATOR).map { |path| File.join(path, 'charm') }.each { |path| File.delete(path) if File.exist?(path) && File.readlines(path).grep(%r{# see com.intellij.idea.SocketLock for the server side of this interface}).any? }
+  end
+
+  zap trash: [
+               "~/Library/Application Support/PyCharm#{version.major_minor}",
```
Comment (vitorgalvao) [inline Casks/pycharm-with-anaconda-plugin.rb:19]: Use `*` instead of interpolating `version`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/pycharm-with-anaconda-plugin.rb (hunk 1 above)
Comment (sicruse) [reply Casks/pycharm-with-anaconda-plugin.rb:12]: unfortunately, yes it is. Thanks for checking though!

File: Casks/pycharm-with-anaconda-plugin.rb (hunk 2 above)
Comment (sicruse) [reply Casks/pycharm-with-anaconda-plugin.rb:19]: Thanks for the feedback @vitorgalvao - so you think:

```suggestion
  zap trash: [
               "~/Library/Application Support/PyCharm*",
```

correct?

Comment (sicruse) [reply Casks/pycharm-with-anaconda-plugin.rb:19]: I have updated the zap stanza in the latest commit.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**